gax-grpc/BUILD.bazel

```diff
@@ -0,0 +1,81 @@
+load("//:gax_java.bzl", "java_tests", "google_java_format", "google_java_format_verification")
+
+_JAVA_COPTS = [
+    "-source",
+    "1.7",
+    "-target",
+    "1.7",
+]
+
+_COMPILE_DEPS = [
```

Comment:
Why do we list dependencies here? Aren't we pulling them from the `dependencies.properties` file?

Reply:
Sorry I think this was just a misunderstanding on my part.